### ISSUE TITLE
Seems a problem with QGuiApplication

### ISSUE THREAD

**Issue by womue1** [yjg30737/pyqt-openai#1]
AFter checking all module installation I found the following problem while running python main.py

main.py", line 21, in <module>
    QGuiApplication.setHighDpiScaleFactorRoundingPolicy(Qt.HighDpiScaleFactorRoundingPolicy.Round)
AttributeError: type object 'QGuiApplication' has no attribute 'setHighDpiScaleFactorRoundingPolicy'

**Comment by yjg30737** [comment]
I've just checked that out and i figure out that `setHighDpiScaleFactorRoundingPolicy` is supported sinse Qt 5.14.

I think we need `setHighDpiScaleFactorRoundingPolicy` because that handle the changing of the Windows scale factors.

For example, if you change the Windows scale factors and don't use the `setHighDpiScaleFactorRoundingPolicy`, Qt window GUI might be broken. So it is pretty important one.

Bottom line, you need to upgrade the Qt version. I will add it to the Requirements section of the readme as well.

You can see more information of it in <a href="https://doc.qt.io/qt-6/qt.html#HighDpiScaleFactorRoundingPolicy-enum">here</a>, Qt documentation.

**Comment by womue1** [comment]
Cool thanks,

PyQt5                         5.15.9
PyQt5-Qt5                     5.15.2

I have the one higher?

**Comment by yjg30737** [comment]
Of course. 5.15.9 is higher one than 5.14 🙂
